### DISCUSSION TITLE
Define default entrypoint for each role

### DISCUSSION THREAD

**Discussion by bernatvadell** [directus/directus#6435] [Feature Requests]
Hi!

I've a role with a very limited access.

I've enabled two modules navigation:
![image](https://user-images.githubusercontent.com/17298604/122968635-0a806180-d38c-11eb-993d-8ffd61c35de2.png)

When the user accesses, by default, it loads "/admin/collections"

In this case, I would like to be able to indicate which is the default "module navigation", in this case, "calendar".

The proposal would be to include a flag/boolean to determine which is the default module or below the list of modules, include a selector to indicate which is the default module.

If you agree with the proposal, I can do the PR.

**Comment by rijkvanzanten** [comment]
I think we could make this a "homepage path" field on the role level, which in turn will make the app use "Last page"  if null, and the specific path if configured. This would allow you to set any module or specific item as the default page, instead of just the module "home" 

@benhaynes yeehaw?

**Reply by benhaynes** [reply]
Absolutely. Especially with the upcoming Insights dashboards, it will be great to offer move control over the initial page route.

**Reply by xu4wang** [reply]
I like the homepage path idea  @rijkvanzanten 

before we have that, can I set "last page" for the user log in the first time?  

thanks

**Reply by xu4wang** [reply]
Maybe with a hook to set last page if it's null in collection direct_users?

**Reply by younky-yang** [reply]
I have the same requirement, the customize homepage is a good idea.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
